ionics_fits/models/laser_rabi.py

```diff
@@ -38,9 +45,12 @@ class LaserFlop(base_class):
 
         This class does not support fitting directly; use one of the subclasses instead.
 
-        The model requires that the system starts out entirely in one of the ground or
-        excited states, specified using :meth:`__init__`'s :param:`start_excited`
-        parameter.
+        The model requires that the internal part of the system starts out
+        entirely in one of the ground or excited states, specified using
+        :meth:`__init__`'s :param:`start_excited` parameter. It further assumes
+        that the motional part of the system starts out as a statistical mixture
```

Comment:
@AUTProgram is `statistical mixture` correct here? Coherent superpositions are fine as well, right?

Reply:
(i.e. it's more that the coherences don't impact the dynamics, than that we can't represent them - as written I would read that to say that coherent states aren't valid).

Can we just say `starts out with a distribution described by`